gradle.properties

```diff
@@ -4,23 +4,23 @@
 pluginGroup = intellij
 pluginName = intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.1.1
+pluginVersion = 0.1.2
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
 pluginSinceBuild = 213
-pluginUntilBuild = 221.*
+pluginUntilBuild = 222.*
```

Comment:
One option might also be to set it to something like `999.*` so in the future people can use the plugin with EAP builds and don't need to wait for a release. There is [one place in the plugin](https://github.com/bufbuild/intellij-buf/blob/1f9ba857d6d9efd0ea0bb062c0336754c5fd2ef1/src/main/kotlin/com/github/bufbuild/intellij/annotator/BufAnalyzeUtils.kt#L37) that uses some tricky APIs but I talked with JetBrains folks and there is no other way at the moment. So this tricky might get broken at some point but then it will be reported and we can do the proper uptade.

Reply:
Let's do that. hah.